public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -955,10 +955,13 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
     /**
      *
      * @param panoramaId
+     * @param force: force to change pano, even if walking is disabled
      * @returns {setPano}
      */
-    self.setPano = function (panoramaId) {
-        if (!status.disableWalking) {
+    self.setPano = function (panoramaId, force) {
```

Comment:
You haven't updated *all* the places where the setPano() method is called since you changed the signature of the method.